pkg/cloudprovider/providers/openstack/openstack_test.go

```diff
@@ -268,7 +268,7 @@ func TestVolumes(t *testing.T) {
 	tags := map[string]string{
 		"test": "value",
 	}
-	vol, err := os.CreateVolume("kubernetes-test-volume-"+rand.String(10), 1, &tags)
+	vol, err := os.CreateVolume("kubernetes-test-volume-"+rand.String(10), 1, "fast", "nova", &tags)
```

Comment:
The test should use "" and "" as zone and volume type, you don't know on what OpenStack instance is this running and what volume types and zones are available there.